app/main.py

```diff
@@ -1 +1,29 @@
-# write your code here
+class Animal:
+    alive = []
```

Comment:
The `alive` list is defined as a class variable, which is correct. However, ensure that this list is managed properly to avoid memory leaks or incorrect references if animals are frequently created and destroyed.